src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+const addBtn = document.querySelector('#add-btn');
+const todoList = document.querySelector('.todo-list');
+const inputField = document.querySelector('#input-field');
+const clearAll = document.querySelector('.clearAll');
+
+const todoListArray = [];
+
+const addnew = () => {
+  todoListArray.push(inputField.value);
+  const div = document.createElement('div');
+  div.classList.add('div-active');
+  div.innerHTML = inputField.value;
+  todoList.insertBefore(div, clearAll);
+  inputField.value = '';
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  div.appendChild(checkbox);
+  checkbox.classList.add('checkbox');
+  checkbox.addEventListener('click', (e) => {
+    e.target.parentElement.classList.add('line-through');
+  });
+  checkbox.addEventListener('dblclick', (l) => {
+    l.target.parentElement.classList.remove('line-through');
+  });
+  clearAll.classList.remove('dis-none');
+};
```

Comment:
- [ ] Your `addnew` function adds a new item even if it's empty. Please some validation to make sure empty to-do items are not added.
- [ ] JavaScript uses the `camelCase` conventionally, please keep it consistent in your project; `addnew` is better as `addNew` by this convention. Consider renaming it accordingly.